src/testSimulator/Classroom.java

```diff
@@ -0,0 +1,117 @@
+package testSimulator;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Classroom {
+
+	private int BStudents;
```

Comment:
What exactly does it represents? I suppose "number of". In that case rename all those variables.